CI/build-mudlet-for-windows.sh

```diff
@@ -123,50 +121,58 @@ echo "LUA_PATH is: ${LUA_PATH}"
 echo "LUA_CPATH is: ${LUA_CPATH}"
 echo ""
 
-if [[ "${MUDLET_VERSION_BUILD,,}" == *"-testing"* ]]; then
-    # The updater is not helpful in this environment (PR testing build)
-    export WITH_UPDATER="NO"
-else
-    # Tagged build, this is a release or a PTB build, include the updater
-    export WITH_UPDATER="YES"
-fi
+echo "Running CMake to configure project ..."
+echo ""
 
 # Since we have this already installed as a package there is no need to build
-# it from a submodule:
+# it from a submodule - set as environment variable for CMake
 export WITH_OWN_QTKEYCHAIN=NO
 
-echo "Running qmake to make MAKEFILE ..."
-echo ""
-
-if [ "${MSYSTEM}" = "MINGW64" ]; then
-    qmake6 ../src/mudlet.pro -spec win32-g++ "CONFIG-=qml_debug" "CONFIG-=qtquickcompiler"
+# Set updater flag based on build type
+if [[ "${MUDLET_VERSION_BUILD,,}" == *"-testing"* ]]; then
+  # The updater is not helpful in this environment (PR testing build)
+  export WITH_UPDATER=NO
 else
-    qmake ../src/mudlet.pro -spec win32-g++ "CONFIG-=qml_debug" "CONFIG-=qtquickcompiler"
+  # Tagged build, this is a release or a PTB build, include the updater
+  export WITH_UPDATER=YES
 fi
 
-echo " ... qmake done."
-echo ""
+# Configure CMake with ccache and Release build type
+CMAKE_ARGS=(
+  -G Ninja
+  -DCMAKE_BUILD_TYPE=Release
+  -DCMAKE_PREFIX_PATH="${MINGW_INTERNAL_BASE_DIR}"
+  -DCMAKE_RUNTIME_OUTPUT_DIRECTORY="${GITHUB_WORKSPACE}/build-${MSYSTEM}/release"
```

Comment:
Is CMake the same as QMake in that a "debug" build will put the "output" in a `/debug` sub-directory instead of this `/release` one?